File: JAVA BASIC WEB - EXERCISES/09. Software-Technologies-Java-Syntax-Basic-Web-Calculator-Skeleton/src/main/java/com/softuni/entity/Calculator.java
package com.softuni.entity;

public class Calculator {
    private double leftOperand;
    private double rightOperand;
    private String operator;

    public Calculator(double leftOperand, double rightOperand, String operator) {
        this.leftOperand = leftOperand;
        this.rightOperand = rightOperand;
        this.operator = operator;
    }

    public double getLeftOperand() {
        return leftOperand;
    }

    public double getRightOperand() {
        return rightOperand;
    }

    public String getOperator() {
        return operator;
    }

    public void setLeftOperand(double leftOperand) {
        this.leftOperand = leftOperand;
    }

    public void setOperator(String operator) {
        this.operator = operator;
    }

    public void setRightOperand(double rightOperand) {
        this.rightOperand = rightOperand;
    }

    public double calculateResult() {
         double result = 0;
         switch (this.operator) {
             case "+":
                 result = this.leftOperand + this.rightOperand;
                 break;
             case "-":
                 result = this.leftOperand - this.rightOperand;
                 break;
             case "*":
                 result = this.leftOperand * this.rightOperand;
                 break;
             case "/":
                 result = this.leftOperand / rightOperand;
                 break;
         }
         return result;
    }
}

File: JAVASCRIPT EXPRESSJS OVERVIEW - EXERCISES/Calculator/models/Calculator.js
function Calculator(leftOperand, operator, rightOperand) {
    this.leftOperand = leftOperand;
    this.operator = operator;
    this.rightOperand = rightOperand;

    this.calculateResult = function () {
        let result = 0;
        switch (operator) {
            case "+":
                result = leftOperand + rightOperand;
                break;
            case "-":
                result = leftOperand - rightOperand;
                break;
            case "*":
                result = leftOperand * rightOperand;
                break;
            case "/":
                result = leftOperand / rightOperand;
                break;
        }
        return result;
    };

}
module.exports = Calculator;
